arm-compute/2017-03-30/swagger/disk.json

```diff
@@ -0,0 +1,1161 @@
+        },
+        "imageReference": {
+          "$ref": "#/definitions/ImageDiskReference",
+          "description": "Disk source information."
+        },
+        "sourceUri": {
+          "type": "string",
+          "description": "If creationOption is Import, this is the URI of a blob to be imported into a managed disk."
+        },
+        "sourceResourceId": {
+          "type": "string",
+          "description": "If createOption is Copy, this is the ARM id of the source snapshot or disk."
+        }
+      },
+      "required": [
+        "createOption"
+      ],
+      "description": "Data used when creating a disk."
+    },
+    "ImageDiskReference": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "A relative uri containing either a Platform Imgage Repository or user image reference."
+        },
+        "lun": {
+          "type": "integer",
+          "format": "int32",
+          "description": "If the disk is created from an image's data disk, this is an index that indicates which of the data disks in the image to use. For OS disks, this field is null."
+        }
+      },
+      "required": [
+        "id"
+      ],
+      "description": "The source image used for creating the disk."
+    },
+    "GrantAccessData": {
+      "properties": {
+        "access": {
+          "type": "string",
+          "enum": [
+            "None",
+            "Read"
+          ],
+          "x-ms-enum": {
+            "name": "AccessLevel",
+            "modelAsString": false
+          }
+        },
+        "durationInSeconds": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Time duration in seconds until the SAS access expires."
+        }
+      },
+      "required": [
+        "access",
+        "durationInSeconds"
+      ],
+      "description": "Data used for requesting a SAS."
+    },
+    "AccessUri": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AccessUriOutput",
+          "description": "Operation output data (raw JSON)"
+        }
+      },
+      "description": "A disk access SAS uri."
+    },
+    "AccessUriOutput": {
+      "properties": {
+        "output": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AccessUriRaw",
+          "description": "Operation output data (raw JSON)"
+        }
+      },
+      "description": "Azure properties, including output."
+    },
+    "AccessUriRaw": {
+      "properties": {
+        "accessSAS": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A SAS uri for accessing a disk."
+        }
+      },
+      "description": "This object gets 'bubbled up' through flattening."
+    },
+    "Snapshot": {
+      "properties": {
+        "sku": {
```

Comment:
this is referencing DiskSku, which has "description": "Gets or sets the disks and snapshots sku name. Can be Standard_LRS or Premium_LRS."